plugins/woocommerce/src/Blocks/BlockTypes/MiniCartItemsBlock.php

```diff
@@ -97,6 +101,13 @@ protected function render_experimental_iapi_markup( $attributes, $content, $bloc
 										<div data-wp-bind--hidden="!state.isAvailableOnBackorder" class="wc-block-components-product-badge wc-block-components-product-backorder-badge" hidden>
 											<?php echo esc_html( $available_on_backorder_label ); ?>
 										</div>
+										<div 
+											class="wc-block-components-product-badge wc-block-components-product-low-stock-badge"
+											data-wp-bind--hidden="!state.isLowInStockVisible"
+											data-wp-text="state.lowInStockLabel"
+											hidden
```

Comment:
I don't think this `hidden` is necessary, as it is inside the `<template>` element. The `data-wp-bind--hidden` attribute would be added during either server-side or client-side rendering.

Reply:
I've removed the `hidden` property and opened https://github.com/woocommerce/woocommerce/issues/59477 to track and discuss the server initialization.